Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ResourceString/ResourceStringXaml.bind

```diff
@@ -0,0 +1,62 @@
+<Page
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages.ResourceString"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:core="using:Microsoft.Xaml.Interactions.Core"
+    xmlns:ui="using:Microsoft.Toolkit.Uwp.UI"
+    mc:Ignorable="d">
+    <Page.Resources>
+        <SolidColorBrush x:Key="BrushRed" Color="Red" />
+        <SolidColorBrush x:Key="BrushBlue" Color="Blue" />
+    </Page.Resources>
+    <Grid>
+        <StackPanel Spacing="8"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center">
+            <TextBlock Text="{ui:ResourceString Name=ResourceStringMarkupTitle}"
+               FontSize="30em"
+               TextWrapping="Wrap"
+               Style="{StaticResource BodyTextBlockStyle}"/>
+
+            <TextBlock Text="With the ResourceString Markup Extension, there's no need to use the Uid appoach to get strings from resource files. Simply call the resource by name in any property that takes a string!"
+                       FontSize="20em"
+                       Margin="20"
+                       TextWrapping="Wrap"
+               Style="{StaticResource BodyTextBlockStyle}"/>
+
+            <TextBlock
+               TextWrapping="Wrap"
+               Style="{StaticResource BodyTextBlockStyle}">
+                <Run>Get a greating from the English resource context: </Run>
+                <Run Text="{ui:ResourceString Name=HelloText, Language=en-US}"></Run>
+            </TextBlock>
+            <TextBlock
+               TextWrapping="Wrap"
+               Style="{StaticResource BodyTextBlockStyle}">
+                <Run>Or get a greeting from the Spanish resource context: </Run>
+                <Run Text="{ui:ResourceString Name=HelloText, Language=es-ES}"></Run>
+            </TextBlock>
+
+            <TextBlock
+               TextWrapping="Wrap" Margin="0,20,0,0"
+               Style="{StaticResource BodyTextBlockStyle}">
+                You can alternatively get a greeting using x:Bind for optimized compiled bind:
+            </TextBlock>
+            <TextBlock
+               TextWrapping="Wrap" Margin="10,10,0,0"
+               Style="{StaticResource BodyTextBlockStyle}">
+                {x:Bind ui:ResourceStringExtension.GetValue('HelloText', 'es-ES')}
```

Comment:
Thanks for checking this out... I'll test it out again asap (likely Monday) and update.

Reply:
@XAML-Knight I tried this again... and it works in my own sample app (Don't use the toolkit sample for this, it is known not to support compiled funcs right now) 
I DID discover a related bug though... you can't leave off the default param 'Language' to get the OS default language. This is not supported in x:Bind funcs. So I've added a new static method to support this.

I will ask another teammate to test it too, but if you can tell me how you tested it I'd love to know why we have differing results.

Reply:
I was able to get this to work in a blank UWP app.

Reply:
Looks like the current code is displaying the syntax for the x:Bind. The XAML panel on the right shows the syntax to the user, we should use this x:Bind to display the result in `TextBlock.Text`, rather than displaying the raw syntax.